### PR TITLE
[WIP] Create a Spire microsite

### PR DESCRIPTION
**Pull request by tixxit**
The goal of this PR is to basically copy in most of our GUIDE into a new microsite. This is largely cribbing from @sritchie 's great work on Algebird's microsite.

### CODE REVIEW

File: docs/src/main/tut/numbers/algebraic.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+layout: docs
+title:  "Algebraic"
+section: "numbers"
+source: "core/shared/src/main/scala/spire/math/Algebraic.scala"
+scaladoc: "#spire.math.Algebraic"
```
Comment (sritchie-stripe) [inline docs/src/main/tut/numbers/algebraic.md]: I cribbed this style from Cats - as far as I can tell, neither they nor the algebird microsite are using this info now, but I think it'd be great to have a custom Jekyll layout that added links to these items.

File: docs/src/main/tut/numbers/real.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+```tut:book
+import spire.math.Real
+import Real.{sin, cos}
+
+def circle(a: Real): Real = (cos(a).pow(2) + sin(a).pow(2)).sqrt
+```
+
+Regardless of the argument we pass to `circle`, it'll return 1:
+
+```tut:book
```
Comment (sritchie-stripe) [inline docs/src/main/tut/numbers/real.md]: :boom: !

File: docs/src/main/tut/authors.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+## Authors
```
Comment (sritchie) [inline docs/src/main/tut/authors.md]: you probably want to add this copied file to .gitignore

File: docs/src/main/tut/contributing.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
+## Contributing to Spire
```
Comment (sritchie) [inline docs/src/main/tut/contributing.md]: same with this

**Comment by tixxit** [comment]
I've moved over the GUIDE now. I'm going to see if I can get the API publishing w/ unidoc as well.

**Comment by non** [comment]
This looks great. Thanks @tixxit.

Do you have a sense of how much more is needed before we merge this?

**Comment by tixxit** [comment]
@non There is some more stuff I want to do, but that can go in a different PR. I think for this one, the most important bit is just testing it out. Let me give it a go tomorrow!

**Comment by denisrosset** [comment]
Hi @tixxit, should we merge this ASAP before the GUIDE and the microsite diverge too much?

**Comment by non** [comment]
Unfortunately this site isn't compatible with SBT 1.0. I'm closing this and will use tut for now.

**Pull request closed without merging**